drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/AttributeColumnPlugin.java

```diff
@@ -79,7 +79,7 @@ public String getAttribute() {
         return attribute;
     }
 
-    public void setAttribute(String attribute) {
+    public void setAttribute(String  attribute) {
```

Comment:
@jomarko This change should be discarded, in my opinion (it adds an useless space)

Reply:
Sure, will do in new commit, thank you